### PR TITLE
Replaces melons

### PR DESCRIPTION
**Pull request by Amelia0010**
With watermelons

### Intent of your Pull Request

Adds water to watermelons, cause you know, watermelons?

Also, the nutriment in watermelons has been halfed. 
#### Changelog

:cl:    
tweak: watermelons now contain water  
tweak: watermelons now contain half the nutriment
/:cl:

### CODE REVIEW

**Comment by X-TheDark** [comment]
REEEEEEEEEEEalism-creep

**Comment by Ratvar** [comment]
Buffs botany chemical reactions tremendiusly cause they can mix potassium-water bombs and mannitol to heal braindamage from meth and earthblood

**Comment by KrabSpider** [comment]
Potassium water grenades, mannitol. Oh yeah, this is probably going to bite us in the ass.

**Comment by Amelia0010** [comment]
yeah but **WATER** melons

**Comment by X-TheDark** [comment]
`Buffs botany chemical reactions tremendiusly cause they can mix potassium-water bombs and mannitol to heal braindamage from meth and earthblood`

Wouldn't this make 0 effective change to botany? They always had sinks/water_tanks/buckets to use water.

File: code/modules/hydroponics/grown/melon.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
 	icon_dead = "watermelon-dead"
 	mutatelist = list(/obj/item/seeds/watermelon/holy)
-	reagents_add = list("vitamin" = 0.08, "nutriment" = 0.4)
+	reagents_add = list("vitamin" = 0.08, "nutriment" = 0.2, "water" = 0.4)
```
Comment (X-TheDark) [inline code/modules/hydroponics/grown/melon.dm:14]: 🚓 UNDOCUMENTED CHANGE DETECTED : Decreased nutriment value from 0.4 to 0.2 🚓 

Botany literally destroyed.

**Comment by Ratvar** [comment]
They cannot inject water into every plant they grow, it takes shitton of time-and healing chems are used on a go in combat situation,commonly .
Neither you can inject water with potassium that easy.

File: code/modules/hydroponics/grown/melon.dm (hunk 1 above)
Comment (Amelia0010) [reply code/modules/hydroponics/grown/melon.dm:14]: I should probably put this in the op, yeah. I changed it to 2 as that's what it is in holy melons with the holy water amount

**Pull request merged**